pkg/codegen/dotnet/gen.go

```diff
@@ -2391,10 +2394,6 @@ func generateModuleContextMap(tool string, pkg *schema.Package) (map[string]*mod
 		cfg.namespaceName = fmt.Sprintf("%s.%s", cfg.RootNamespace(), namespaceName(infos[pkg].Namespaces, pkg.Name))
 	}
 
-	visitObjectTypes(pkg.Config, func(t *schema.ObjectType) {
-		getModFromToken(t.Token, pkg.Reference()).details(t).outputType = true
-	})
-
```

Comment:
This part here is potentially a breaking change ⚠️ object types referenced from config variables were marked as outputs and were generated inside the `Outputs` namespace. This change removes them from there